### ISSUE TITLE
Cannot review last character in nonfocusable objects in Windows 10's Settings

### ISSUE THREAD

**Issue by tmthywynn8** [nvaccess/nvda#8755]
When using the review cursor to navigate by character in some dialogs in Windows 10, e.g., Settings (Windows+I) or network flyouts (access through the notification area), NVDA will not allow you to reach the last character in objects that cannot be navigated to with the system focus. This can be problematic if, for example, you are trying to read a bluetooth confirmation code to enter or confirm on a remote device, e.g, a keyboard or braille display.

### Steps to reproduce:
1. Open Settings with Windows+I.
2. If the focus does not land on the "Search box, Find a setting" edit-box, use the Tab key to move to it.
3. With simple review mode enabled, use the "move to previous object" command&mdash;you should land on an object that says "Windows Settings".
4. Use the "move to next character in review" command to navigate to the last character.

### Actual behavior:
NVDA reports that `g' is the last character. Note that using the "move to end of line in review" command will not exhibit this behavior.

### Expected behavior:
NVDA should be reporting that `s' is the last character.

#### NVDA Installed/portable/running from source:
Installed

#### NVDA version:
alpha-16026,4730a9f3

#### Windows version:
Windows 10 Home Version 1709 (OS Build 16299.611)

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
Almost all official releases of NVDA since Windows 10 was released.

**Comment by josephsl** [comment]
Hi, something I’ve been thinking of reporting for a long while no unless this is a dulicatew. CC @MichaelDCurran

**Comment by Adriani90** [comment]
Closing as duplicate of #8093.
